doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -118,45 +125,70 @@ Time
 
 The proper operation of the P-GPS subsystem depends on an accurate sense of time.
 For use cases where a cloud connection can be established easily, use the :ref:`lib_date_time` library with NTP enabled.
-Otherwise, a battery-backed real-time clock calendar chip must be used so that accurate time is available regardless of cloud availability after reset.
+Otherwise, use a battery-backed real-time clock calendar chip so that accurate time is available regardless of cloud availability after reset.
+
+Transport Mechanisms
+********************
+
+Follow these three steps to request and store P-GPS data in the device:
+
+1. Send the request to `nRF Cloud`_
+#. Receive a URL from `nRF Cloud`_ pointing to the prediction set
+#. Download the predictions from the URL
+
+The first two steps use the network transport defined by the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_TRANSPORT` option.
+
+The default configuration selects the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_TRANSPORT_MQTT` option if the :kconfig:option:`CONFIG_NRF_CLOUD_MQTT` option is active.
+MQTT use is built into the P-GPS library.
+
+The library uses REST or other transports by means of the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE` option.
+REST support is not built into the P-GPS library and must be provided instead by the application.
+The :ref:`gnss_sample` sample is one example.
+
+The third step uses the network transport defined by the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT` option.
+
+The P-GPS library uses the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_HTTP` option when using MQTT for the main transport.
+Applications that use REST for the main transport usually also use the HTTP option for the download transport.
+This download transport is built into the P-GPS library.
+
+Alternatively, use the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_CUSTOM` with :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE` to manage the full flow of data outside of the P-GPS library.
+Call the following functions when using this configuration:
+
+1. :c:func:`nrf_cloud_pgps_begin_update`
+#. :c:func:`nrf_cloud_pgps_process_update`
+#. :c:func:`nrf_cloud_pgps_finish_update`
```

Comment:
```suggestion

* :c:func:`nrf_cloud_pgps_begin_update`
* :c:func:`nrf_cloud_pgps_process_update`
* :c:func:`nrf_cloud_pgps_finish_update`
```

Reply:
I used a numbered list because the steps should be executed in that order.